hack/run_with_local_webhook.sh

```diff
@@ -15,11 +15,12 @@ TMPDIR=${TMPDIR:-"/tmp/k8s-webhook-server/serving-certs"}
 SKIP_CERT=${SKIP_CERT:-false}
 CRC_IP=${CRC_IP:-$(/sbin/ip -o -4 addr list crc | awk '{print $4}' | cut -d/ -f1)}
 FIREWALL_ZONE=${FIREWALL_ZONE:-"libvirt"}
+WEBHOOK_PORT=${WEBHOOK_PORT:-${WEBHOOK_PORT}}
```

Comment:
shouldn't the default here be `9443` instead of `WEBHOOK_PORT`?

Reply:
ack, I see the same pattern used elsewehere https://github.com/openstack-k8s-operators/openstack-operator/blob/5daf17afd0624a45fcdee80c0bb8bbe00e939487/hack/run_with_local_webhook.sh#L17